src/_zkapauthorizer/_plugin.py

```diff
@@ -129,13 +130,22 @@ def get_storage_server(
                 kwargs.pop(u"ristretto-signing-key-path"),
             ),
         )
+        public_key = PublicKey.from_signing_key(signing_key)
         announcement = {
             u"ristretto-issuer-root-url": root_url,
+            u"ristretto-public-keys": public_key.encode_base64(),
```

Comment:
Looking at this again, it should be a list. I'll update it to be so.

Reply:
I think this doesn't do anything in practice though, since this announcement is only used if the server connects to introducers, which we only do in the nix system tests.